src/webWidget/models.ts

```diff
@@ -146,6 +147,8 @@ export type WebWidgetSignParams = {
   }
 }
 
+export type WebWidgetKeyExportParams = {}
```

Comment:
I think we need to pass an access token right?

Reply:
No, this is sent every time.
This interface is only with function parameters. See the other examples

Reply:
Oh, the WebWidgetOreIdOptions are included already?

Reply:
yes